content/latest/api/ysql/commands/ddl_create_domain.md

```diff
@@ -0,0 +1,71 @@
+---
+title: CREATE DOMAIN
+linkTitle: CREATE DOMAIN
+summary: Create a new domain in a database
+description: CREATE DOMAIN
+menu:
+  latest:
+    identifier: api-ysql-commands-create-domain
+    parent: api-ysql-commands
+aliases:
+  - /latest/api/ysql/ddl_create_domain/
+isTocNested: true
+showAsideToc: true
+---
+
+## Synopsis
+
+The `CREATE DOMAIN` command creates a user-defined data type with a range, optional DEFAULT, NOT NULL and CHECK constraint.
```

Comment:
It's not clear what "with a range" means here.
We should also add a line about why domains are needed - for example, "Domains are useful to abstract data types with common constraints. For example, domain can be used to represent phone number columns that will require the same CHECK constraints on the syntax."

Reply:
Hi Rahul, it'd be clearer if you moved the word constraint up.
The command creates a user-defined datatype that has optional constraints such as range of valid values, NOT NULL, CHECK, and DEFAULT.